src/features/create-dao/steps/DAOSummary.module.scss

```diff
@@ -0,0 +1,70 @@
+.Row {
+	display: flex;
+	margin-top: 0.625rem;
+	width: 20.5rem;
+
+	&_Summary {
+		display: flex;
+		margin-top: 0;
+	}
+
+	.Column {
+		display: flex;
+		flex-direction: column;
+		
+		.Heading {
+			margin: 0 0 1rem 0;
+		}
+		
+		.MediaInput {
+			height: 12.5rem;
+			width: 12.5rem;
+			margin-right: 2rem;
+			border-radius: 0.5rem !important;
+		}
+		
+		.Summary {
+			margin-right: 1.3rem;
+			width: 8rem;
```

Comment:
I think these styles can be improved to future proof responsive design, but as we've discussed we can circle back on some of the styling